libs/order/src/facades/order/order-facade.interface.ts

```diff
@@ -27,4 +28,25 @@ export interface DaffOrderFacadeInterface<T extends DaffOrder = DaffOrder> exten
   getPayment$(orderId: T['id']): Observable<T['payment']>;
   getInvoices$(orderId: T['id']): Observable<T['invoices']>;
   getCredits$(orderId: T['id']): Observable<T['credits']>;
+
+  /**
+   * The specified order's grand total.
+   */
+  getGrandTotal$(orderId: T['id']): Observable<DaffOrderTotal>;
+  /**
+   * The specified order's subtotal.
+   */
+  getSubtotal$(orderId: T['id']): Observable<DaffOrderTotal>;
+  /**
+   * The specified order's shipping total.
+   */
+  getShipping$(orderId: T['id']): Observable<DaffOrderTotal>;
```

Comment:
I think this (and the selectors) should be `getShippingTotal$` instead, just to be more clear.

Reply:
done

Reply:
you missed the MockFacade

Reply:
yeah I always forget it